manifest.json

```diff
@@ -47,7 +47,7 @@
   "custom": {
       "gear-builder": {
           "category": "converter",
-          "image": "flywheel/dicom-send:1.1.2"
+          "image": "flywheel/dicom-send:2.0.0"
```

Comment:
Sounds backward compatible. Would bump the minor version only.

Reply:
I bumped major bc it will now fail when it didn't use to, @dpark6060 thoughts?